migration/tests/tests.rs

```diff
@@ -0,0 +1,33 @@
+#![cfg(test)]
+
+use migration::{Migrator, MigratorTrait};
+use test_context::test_context;
+use test_log::test;
+use trustify_common::db::{test::TrustifyContext, ConnectionOrTransaction};
+
+#[test_context(TrustifyContext, skip_teardown)]
+#[test(tokio::test)]
+async fn test_migrations(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
+    let db = ctx.db;
+
+    let conn_or_tx = db.connection(&());
+    let conn = match conn_or_tx {
+        ConnectionOrTransaction::Connection(conn) => conn,
+        _ => {
+            panic!("Error while getting db connection");
+        }
+    };
```

Comment:
You don't really need this...

Reply:
I guess I don't know other way to do.

Reply:
oh , yeah we refactored!
you're awesome!